ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx

```diff
@@ -275,9 +275,11 @@ export const ScenePlayer: React.FC<IScenePlayerProps> = ({
 
       if (curTime != prevCaptionOffset) {
         if (!isDirect) {
-          removeCaptionOffset(player, curTime);
-          prevCaptionOffset = curTime;
-          console.log(`updated prevCaptionOffset to ${prevCaptionOffset}`);
+          if (prevCaptionOffset != 0) {
```

Comment:
This change seems to break the subtitle syncronization for me.
Won't prevCaptionOffset be 0 the first time we seek ?

Reply:
I'm not sure how this got here. But you're right.